### ISSUE TITLE
Current line in editor doesn't get highlighted

### ISSUE THREAD

**Issue by Deon588** [Microsoft/vscode#34835]
I am using the latest vsCode (x64) with the dark high contrast theme.  I have low vision and would like to be able to have the current line in the editor highlighted to make it easier to find the current line.  I remember when I previously used vsCode there was a light outline around the currently focused line.  That outline seems to have disappeared after I updated...

- VSCode Version:
- OS Version:
Windows 10 pro x64
Steps to Reproduce:

1.
switch to the high contrast theme in windows 10
2.
Start the windows magnifier
3.
Open any file in the vsCode editor and navigate up and down using the keyboard

Reproduces without extensions: Yes/No

**Comment by ramya-rao-a** [comment]
@Deon588 I still see the outline on the current line.
I am on Windows 10 and used the High Contrast theme.

![image](https://user-images.githubusercontent.com/16890566/30755137-5b7fb36e-9f7a-11e7-9002-183f21258553.png)


Can you share your settings?
Try with all extensions disabled just to ensure none of the extensions is interfering (Run `code --disable-extensions` from the command line).

Any other OS setting you think I should have to be able to repro this?

**Comment by Deon588** [comment]
@ramya-rao-a How can I share my settings?  I ran vsCode with extensions disabled but still getting the same result.  Your screenshot looks perfect, really hope I can get this to work as I recently moved from using Notepad++ to vsCode and would HATE to have to go back to npp
![screenshot 7](https://user-images.githubusercontent.com/12716437/30762357-08686212-9fe2-11e7-888e-27a87fcf7575.png)
...

**Comment by ramya-rao-a** [comment]
You can open your settings by going through
- File -> Preferences -> Settings
- Or you can run the cmd from the cmd pallet `Preferences: Open User Settings`
- Or use the keyboard shortcut `Ctrl+,`

The left side will have the default settings, and the right side would have the settings you customized.
Copy the latter and paste it here.

cc @alexandrudima

**Comment by Deon588** [comment]
@ramya-rao-a
This is what I have there, doesn't really seem related except maybe for the theme?

{
    "workbench.colorTheme": "Default High Contrast",
    "window.zoomLevel": -2,
    "workbench.startupEditor": "newUntitledFile"
}

**Comment by Deon588** [comment]
I uninstalled/reinstalled vsCode today but still getting the same result...  How can I completely remove vsCode?  I uninstalled and deleted all data from appdata but after I reinstalled it still opened the most recent project...

**Comment by ramya-rao-a** [comment]
You don't need to uninstall VS Code for this. Doesn't look like an installation issue.

Do you have any other OS level high contrast related setting turned on which we can use to repro the issue?

**Comment by Deon588** [comment]
No nothing...  The strange thing is it worked initially (with the 32bit version) I then receved a notification saying the 64bit version is now available.  So maybe I should try re-install the 32bit version?

**Comment by ramya-rao-a** [comment]
That is strange. Yes, do give the 32 bit version a go.

**Comment by Deon588** [comment]
So I re-installed the 32bit version and it works now!!!  So this is definitely a bug in the 64bit version...  Thanks @ramya-rao-a for all your effort/time...

**Comment by Deon588** [comment]
So I re-installed the 32bit version and it works now!!!  So this is definitely a bug in the 64bit version...  I just updated the 32bit version and it broke the line outline again...  So it seems it's both the 32bit and 64bit versions which are affected...
Thanks @ramya-rao-a for all your effort/time...

**Comment by alexandrudima** [comment]
@Deon588 Do you perhaps also use screen reader software? Do you operate VSCode through a Screen Reader, or visually, or both? 

We turn off certain features when we detect a Screen Reader is attached. We indicate that with a green "Screen Reader Detected" in the status bar.

I believe the current line highlight is one of the things we turn off (together with minimap, folding, word wrapping). The problem is that these concepts can not be currently communicated via w3c aria support to a screen reader.

To turn off our reaction to screen readers, you can configure `"editor.accessibilitySupport": "off"`.

**Comment by alexandrudima** [comment]
I have pushed a change that clarifies what happens when we detect a screen reader via #32166 https://github.com/Microsoft/vscode/issues/32166#issuecomment-331932055

**Comment by Deon588** [comment]
That was it...  I am using a screen reader.  Why disable the outline when a screen reader is present though?  It doesn't seem to interfere with the screen reader at all?

**Comment by Deon588** [comment]
@alexandrudima
  I turned accessibility features off as you suggested however now my screen reader can't read the editor text at all...  I know that the outline doesn't interfere with the screen reader at all as I used it in a earlier version without any issues...  Is there now way I can just enable the outline without disabling all other accessibility features?  If not this seems like more of an issue than a resolution...  I have low vision and need to see my current line even if I use a screen reader.  Finding the insertion cursor is MUCH harder for me than finding that big outline which used to be there before this was implemented... @ @ramya-rao-a

**Comment by alexandrudima** [comment]
@Deon588 Makes sense, we can enable the current line highlight, as it does not interfere at all with screen readers.
